.github/codecov.yml

```diff
@@ -2,3 +2,4 @@ codecov:
   require_ci_to_pass: yes
   notify:
     wait_for_ci: yes
+github_checks: false
```

Comment:
Something has changed recently in the default integration with GitHub, and I disabled it.

PRs were prematurely visible as "failed" due to incomplete coverage information.

Also, some IMO annoying inline warnings were polluting the code review page.